### PR TITLE
Next

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-  "dependencies": {
-    "o-grid": "^4.0.0",
-    "o-colors": ">=2.5.0 <4",
-    "o-assets": ">=0.4.1 <4",
-    "o-hoverable": ">=0.1.1 <4",
-    "ftdomdelegate": ">=1.0.0 <3",
-    "o-hierarchical-nav": ">=2.0.0 <4"
+  "resolutions": {
+    "o-ft-icons": "^2.3.4",
+    "o-grid": "^4.0.0"
```
Comment (keirog) [inline bower.json]: Shouldn't need the `o-grid` resolution anymore; not sure about `o-ft-icons` though

File: demos/header.html [hunk 2]
```diff
@@ -1,83 +1,496 @@
 <!DOCTYPE html>
-<html class="o-hoverable-on o-hoverable-on">
+<html class="o-hoverable-on js">
```
Comment (keirog) [inline demos/header.html]: Could this be added via the demo JS instead?

Comment (AlbertoElias) [inline demos/header.html]: When you go to the registry and see the html for the header, you don't see that

Comment (keirog) [inline demos/header.html]: Ah ok

File: bower.json (hunk 1 above)
Comment (AlbertoElias) [inline bower.json]: Do you want me to remove them then?

**Comment by keirog** [comment]
I'm guessing you'll sort the readme out later, so :+1:

File: bower.json (hunk 1 above)
Comment (keirog) [inline bower.json]: Ah yep, still needs `o-ft-icons` resolution for now

**Pull request merged**